src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -155,22 +157,43 @@ def train_preference_comparisons(
             relabel_reward_fn=relabel_reward_fn,
         )
 
-    if trajectory_path is None:
+    if num_agents < 1:
```

Comment:
Once you're at it, you might as well check it's an integer. No reason to handle custom error case for range but then let a "more obscure" error arise if float.